sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -40,12 +42,21 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     }
   },
 
+  showRegisterModal: function(){
+    $("#register-modal").openModal();
+  },
+
+  showForgotPasswordModal: function(){
+    $("#forgot-modal").openModal();
+  },
+
   render: function() {
     this.close();
 
     activeTab = $(".tab .active").text().toLowerCase() || "categories";
 
-    var content = this.template({active: activeTab, current_user: this.user, feeds: this.feeds, categories: this.categories});
+    var content = this.template({active: activeTab, current_user: this.user, feeds: this.feeds,
+                                 categories: this.categories, isLoading: this.loading});
```

Comment:
To fix the padding when the loading goes away, it was very easy to just add a check on isLoading and remove the padding (by applying class=“hide”) if it’s off.